plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -285,6 +285,18 @@ advanced option interactiveLayout: boolean {
     targets parents
 }
 
+advanced option omitNodeMicroLayout: boolean {
+    label "Omit Node Micro Layout"
```

Comment:
This is a "negated" layout option in that setting it to `true` actually _disables_ something. I suggest doing it the other way around: calling it `runNodeMicroLayout` and changing its default to `true`.

Reply:
I felt that - since running micro layout is the default - a user would look for a layout option to _omit_ that behavior.

I'm not sure what is more intuitive to most people though. Yours maybe reduced cognitive load :D